### DISCUSSION TITLE
Real invisibility potions

### DISCUSSION THREAD

**Discussion by 1ByteBit** [PurpurMC/Purpur#1298] [__Dismissed__] [answered]
When drinking an invisibility potion, effect particles shouldn't appear

**Comment by RealRONiN** [comment]
Could be a cool config option, and definitely useful for roleplay servers!

**Comment by katekerllenevich** [comment, marked as answer]
While this would be nice, in my testing when disabling particles from being spawned on the server, the client still went ahead and spawned them anyways

**Reply by 1ByteBit** [reply]
Would it not be possible to utilize what the /effect command does as that command has an option to disable the particles of an effect

**Reply by BillyGalbreath** [reply]
That command argument doesn't actually hide the particles, just makes them more ghostly/transparent and prevents the effect icon on the HUD.
